docs/.vitepress/theme/components/Examples/WCEmbed/webcontainer.ts

```diff
@@ -131,7 +131,7 @@ async function bootWebContainer(terminal: import('xterm').Terminal) {
 
   terminal.write('Booting WebContainer...');
 
-  webcontainer = WebContainer.boot();
+  webcontainer = WebContainer.boot({ workdirName: 'demo' });
```

Comment:
:star: This changes the folder name visible in the terminal from `~/webcontainersio-98a7w9` or whatever to simply `~/demo`.